src/index.html

```diff
@@ -1,5 +1,7 @@
 <!doctype html>
 <html lang="en">
+
+
   <head>
```

Comment:
don't add blank lines between parent and child elements, fix it everywhere
```suggestion
<html lang="en">
  <head>
```